core/deployment/src/main/java/io/quarkus/deployment/images/ContainerImages.java

```diff
@@ -0,0 +1,92 @@
+package io.quarkus.deployment.images;
+
+import io.quarkus.deployment.pkg.builditem.CompiledJavaVersionBuildItem;
+
+/**
+ * This class is used to define the container images that are used by Quarkus.
+ * <p>
+ * For each image, the image name and version are defined as constants:
+ * <p>
+ * - {@code x_IMAGE_NAME} - the name of the image without the version (e.g. {@code registry.access.redhat.com/ubi8/ubi-minimal})
+ * - {@code x_VERSION} - the version of the image (e.g. {@code 8.10})
+ * - {@code x} - the full image name (e.g. {@code registry.access.redhat.com/ubi8/ubi-minimal:8.10})
+ */
+public class ContainerImages {
```

Comment:
This looks good but I think the versions should be at the top so that we have to change only one version.

I would still keep the independent versioning you used (having `UBI8_MINIMAL_VERSION` makes perfect sense) but I would refer to global constants: `UBI8_VERSION`, `UBI9_VERSION` and so on.

Reply:
Done, except for the micro and binary s2i tags are they change really rarely (once in the last 5 years)